src/bg_triangles.c

```diff
@@ -610,16 +788,44 @@ void sub_8007858(u8 param0, int_vcount minY, int_vcount maxY, u16 param3, u16 pa
         minY++;
     }
 }
+
+// NOTE: Has to be below the code for matching...
```

Comment:
Why?

Reply:
It not unlikely, but it's weird that the data survived but not the code referencing it.